pom.xml

```diff
@@ -1176,9 +1176,21 @@
                 <enabled>true</enabled>
             </releases>
         </repository>
+        <repository>
+            <id>xenit-private</id>
+            <url>https://repo.xenit.eu/K8TBkpYRAtTc9flA/private/maven/</url>
```

Comment:
You can use a cleaner URL if you use one based on entitlement token authentication (see https://xenitsupport.jira.com/wiki/spaces/ST/pages/3701047303/How+to+-+Cloudsmith )

Found it at https://cloudsmith.io/~xenit/repos/private/setup/?auth_method=basic_token#formats-maven

```<repositories>
  <repository>
    <id>xenit-private</id>
    <url>https://repo.xenit.eu/basic/private/maven/</url>
    <releases>
      <enabled>true</enabled>
      <updatePolicy>always</updatePolicy>
    </releases>
    <snapshots>
      <enabled>true</enabled>
      <updatePolicy>always</updatePolicy>
    </snapshots>
  </repository>
</repositories>
```

Reply:
Mildly annoying how that setup documentation is hidden away on cs.

Reply:
Build failed due to 401 when accessing the repo for the patched dependency. It would seem GHA is not correctly set up to use this format?

Reply:
Yes, annoying indeed.

Secrets CLOUDSMITH_APIKEY and CLOUDSMITH_USER are available. I would ask on Slack.
(If this is too much of a detour, you can do it later. It's only of low importance).